app/assets/stylesheets/home.scss

```diff
@@ -39,6 +39,6 @@
   }
 
   input:hover {
-    color: #abe6ff;
+    color: #ffd298;
```

Comment:
Color literals like `#ffd298` should only be used in variable declarations; they should be referred to via variable everywhere else.